pontoon/api/urls.py

```diff
@@ -10,8 +10,8 @@
 
 urlpatterns = [
     # GraphQL endpoint. In DEV mode it serves the GraphiQL IDE if accessed with Accept: text/html
-    url(
-        r"^graphql",
+    path(
+        "graphql",
```

Comment:
Apparently this change breaks URLs like https://pontoon.mozilla.org/graphql/.

I guess if we add slash at the end, both https://pontoon.mozilla.org/graphql and https://pontoon.mozilla.org/graphql/ will work, which would also be consistent with other URL patters across the app.

Reply:
If we have the slash at the end, requests to https://pontoon.mozilla.org/graphql would be redirected to https://pontoon.mozilla.org/graphql/, which might break existing workflows/scripts. As far as I can tell, the URL endpoint without the slash is canonical for GraphQL (e.g., see the [graphene-django docs](https://docs.graphene-python.org/projects/django/en/latest/installation/#id1)), and is already used in scripts (e.g., [here](https://github.com/flodolo/scripts/blob/954fa857af8422b71662470106415924e4524800/mozilla_l10n/cldr_data/cldr_coverage.py)).
Is https://pontoon.mozilla.org/graphql/ actually used, except for the two JS files in the frontend that I modified?

I see two options here:
1. https://pontoon.mozilla.org/graphql is the only endpoint: `path("graphql", ...)`
2. Both versions with and without slash reach the endpoint: `re_path(r"^graphql/?$", ...)` 

What do you think?

Reply:
Thanks for clarifying!

Another important user of the API is the Pontoon Add-on, which also doesn't use the slash:
https://github.com/MikkCZ/pontoon-addon/blob/d8d023788444a698f0cab37ac8d41caecd3aa3d9/src/packages/background/src/RemotePontoon.js#L139

Stas also used links without slash in the blog post:
https://blog.mozilla.org/l10n/2017/10/10/exposing-pontoon-data-through-api/

Since we don't have a lot of other API consumers ATM, let's move on with this change.